rattestation/attestation.c

```diff
@@ -51,26 +60,184 @@ convert_bin_to_hex_new(const uint8_t *bin, int length)
 	len = MUL_WITH_OVERFLOW_CHECK(len, sizeof(char));
 	len = ADD_WITH_OVERFLOW_CHECK(len, 1);
 	char *hex = mem_alloc0(len);
+	if (!hex) {
+		FATAL("ERROR: Failed to allocate memory");
+	}
```

Comment:
mem_alloc0() already does this check internally. It triggers an assertion and fatally terminates.
Just remove this check.

Reply:
Done